lib/fencing.py.py

```diff
@@ -711,10 +711,16 @@ def check_input(device_opt, opt, other_conditions = False):
 		options["--action"] = "off"
 
 
-	if options["--action"] == "validate-all" and not other_conditions:
-		if not _validate_input(options, False):
```

Comment:
I think it'd be better to make a variable called e.g. validate_format to use here, so we can skip the elif part below.

Reply:
I like the idea of having validate_actions. But it won't help us with if/elif part, the code still have to be branched.